test/functionalTest/cases/1005_GET_v2_entities_string/1005_GET_v2_entities_string.test

```diff
@@ -0,0 +1,137 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I should  have though in four digits numbers when we start numbering functional test cases.... Probably an overall renaming (e,g. 766_XXX to 0766_X) now would make sense (although not as part of this PR, of course).

Or maybe I'm just delaying the problem and in a couple of month we would reach issue number #10000 :D

NTC